code/datums/gamemodes/campaign/loadout_items/SOM/secondaries.dm

```diff
@@ -139,7 +139,7 @@
 
 /datum/loadout_item/secondary/kit/som_corpsman/advanced
 	name = "Advanced meds"
-	desc = "A variety of advanced medical injectors including Russian Red, as well as rezadone, a powerful chemical able to treat genetic damage in humanoids."
+	desc = "A variety of advanced medical injectors including Russian Red, rezadone and Re-Grow, allowing for the treatment cloneloss and missing limbs."
```

Comment:
treatment of cloneloss

Reply:
ye?

Reply:
as in the word 'of' is missing right now

Reply:
oh @Arturlang just make a change suggestion nerd, I can't read.